configure.ac

```diff
@@ -388,10 +380,58 @@ AS_IF([test "x$NETSTAT_PATH" != "x" ], [
     AC_MSG_NOTICE([NETSTAT_PATH...] $NETSTAT_PATH)
 ])
 
+# use resolvconf if present
+AC_PATH_PROG(RESOLVCONF_PATH, [resolvconf], [DISABLED], "$PATH:/sbin:/usr/sbin")
+
+# allow override at configure time
+AC_ARG_WITH([resolvconf],
+        AS_HELP_STRING([--with-resolvconf],
+                       [Set the path to the resolvconf executable]),
+        RESOLVCONF_PATH="$withval"
+)
+
 AC_SUBST(RESOLVCONF_PATH)
 AS_IF([test "x$RESOLVCONF_PATH" != "x" ], [
     AC_MSG_NOTICE([RESOLVCONF_PATH...] $RESOLVCONF_PATH)
 ])
+AS_IF([test "x$RESOLVCONF_PATH" = "xDISABLED"], [
+	AC_DEFINE(HAVE_RESOLVCONF, 0)
+	AC_MSG_NOTICE([HAVE_RESOLVCONF... 0])
+],[
+	AC_DEFINE(HAVE_RESOLVCONF, 1)
+	AC_MSG_NOTICE([HAVE_RESOLVCONF... 1])
+])
+
+# the default for the --use-resolvconf runtime command line option
+AC_ARG_ENABLE([resolvconf],
+	AS_HELP_STRING([--enable-resolvconf],
+			[Enable usage of resolvconf by default, \
+			use --disable-resolvconf for the opposite]))
+
+# Checks for existence of specific files if not cross-compiling.
+AS_IF([test "x$enable_resolvconf" = "x" ], [
+  AC_CHECK_FILE([$RESOLVCONF_PATH],[
+    AS_IF([$RESOLVCONF_PATH -l &>/dev/null], [
```

Comment:
Perhaps document the target distributions where you would expect `resolvconf -l` to return an exit status of 0?

Reply:
done